recipes/cutde/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "cutde" %}
+{% set version = "21.8.20" %}
+{% set sha = "b804249cf138946c110abb7bb93e5557ae457c45eb66d1e0660ecdd5664aec94" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cutde-{{ version }}.tar.gz
+  sha256: {{ sha }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

The arguments are not needed anymore explicitly and enforced via conda-build.
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```